build/Publish/PublishNupkgToTransportFeed.targets

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'Common.props'))\Common.props" />
+
+  <PropertyGroup>
+    <!-- Capture 'OutputPath' before Publish.csproj modifies it -->
+    <OutputPathTransportFeed>$(OutputPath)</OutputPathTransportFeed>
+    <BuildTasksFeedDllVersion>1.0.0-prerelease-01929-02</BuildTasksFeedDllVersion>
+    <BuildTasksFeedDll>$(NuGet_Packages)\Microsoft.DotNet.Build.Tasks.Feed\$(BuildTasksFeedDllVersion)\lib\desktop\Microsoft.DotNet.Build.Tasks.Feed.dll</BuildTasksFeedDll>
```

Comment:
Is this necessary?  I suppose it is for this version of the package.  When you update to the later version, you should be able to get rid of this logic and just let dotnet determine the dll and handle imports.

Reply:
Can you elaborate on this comment - I do not think the VSO msbuild-task implementation uses 'dotnet'.

Reply:
Can you send me a link to your official build for sdk so I can get a better idea for how this is executed?